docs/source/api/recipes/recipes.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    pos = xgi.barycenter_spring_layout(H, seed=2)
</pre><p>Can we add a comment here that clearly states this is where the magic is happening? i.e. "#&nbsp;Compute positions for nodes that will be used for all plots"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
Done.